### DISCUSSION TITLE
WebkitSpeechRecognition doesn't work in SafeExamBrowser 3.3 on windows.

### DISCUSSION THREAD

**Discussion by DeveloperAzamat** [SafeExamBrowser/seb-win-refactoring#386] [Q&A]
WebkitSpeechRecognition doesn't work in SafeExamBrowser 3.3 on windows. How can I use it?

**Comment by dbuechel** [comment]
It might be that this could be enabled via a configuration flag of the browser engine. You'd need to investigate this on your own, in case you'll find a solution you could either post it here or create a pull request and we'd try to integrate it into the next upcoming version.

**Reply by DeveloperAzamat** [reply]
We tried, but we can't. If you know how to do this please tell me? When can we expect the next version with webkitSpeechRecognition support? We really need this, thanks.

**Reply by danschlet** [reply]
SEB is a free software which needs community support. As none of the paying contributors of the SEB Alliance requested this feature, it has no priority for us. Check out the CEF/Cef Sharp GitHub pages where you can place an question/create an issue about this feature.

**Reply by DeveloperAzamat** [reply]
CEF / Cef Sharp GitHub looked, it says it needs to be done when building the browser, can you take this into account in the next version? Please, help

**Reply by dbuechel** [reply]
Unfortunately not, we do not create a custom build of the engine but rather use the official CefSharp releases.

**Reply by DeveloperAzamat** [reply]
If we can change CefSharp, how can we integrate it into SafeExam?

**Reply by dbuechel** [reply]
You'll first need to build CefSharp, then reference that custom build in the SEB source code and finally build your own, custom SEB version.
